graphics/jxrlib/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                jxrlib
+version             1.2_git20170615-f752187
+set version_name    1.2~git20170615.f752187
+revision            0
+license             BSD-2-Clause
```

Comment:
I don't think this is a valid license value.  The list of valid licenses can be found here: https://trac.macports.org/wiki/PortfileRecipes#licensekeyword

Looks like this should perhaps just be `BSD`?

Reply:
Thanks done. FWIW, this [MacPorts](https://trac.macports.org/wiki/PortfileRecipes) shows a `BSD-3-Clause`. I just set this to `BSD`.
```sh
TZ=UTC date && port -q info --license --index all | grep -ve ^-- -e '^$' | tr ' ' '\n' | tr -d '{}()' | grep -vxe and -e or -e License: -e '^$' -e '^--.*$' | sort -fu | sed 's/^ */* /'
```